lms/djangoapps/teams/static/teams/js/views/team_join.js

```diff
@@ -0,0 +1,132 @@
+;(function (define) {
+'use strict';
+
+define(['backbone',
+        'underscore',
+        'gettext',
+        'text!teams/templates/team-join.underscore'],
+       function (Backbone, _, gettext, team_join_template) {
+           return Backbone.View.extend({
+
+               errorMessage: gettext("An error occurred. Please try again."),
```

Comment:
@catong This is a generic message and will be shown in cases where we can't show an proper message due to lack of information.